accounts/abi/bind/base.go

```diff
@@ -149,6 +149,25 @@ func DeployContract(opts *TransactOpts, abi abi.ABI, bytecode []byte, backend Co
 	return c.address, tx, c, nil
 }
 
+// Address returns the contract deployment address.
+func (c *BoundContract) Address() common.Address {
+	return c.address
+}
+
+// GetABI returns the contract ABI Object of the BoundContract.
+func (c *BoundContract) GetABI() abi.ABI {
+	return c.abi
+}
+
+// Method retrieves the ABI method with the given name.
+func (c *BoundContract) Method(name string) (*abi.Method, error) {
+	method, exist := c.abi.Methods[name]
+	if !exist {
+		return nil, fmt.Errorf("method '%s' not found", name)
+	}
+	return &method, nil
+}
```

Comment:
In what usecase would this be needed?  The point of the wrappers is to provide an abstraction where callers doesn't have to deal with lowlevel abi, but deal with higher abstractions. 
IF you want low level abi access, then why use bound contracts?

Reply:
Why prevent access to the lower-level abstraction?

In my usecase, I am leveraging abigen and the bound contracts across test suites and in actual implementation. This reduces the amount of code I have to write or reimplement by hand. In my usecase, I have instances where I need to access lower-level events and lower-level methods and already have a bound contract initialized. Instead of reinitializing all of these, why not make them publically accessible if they are already initialized in the bound contracts?